src/main/java/io/appium/java_client/internal/ElementMap.java

```diff
@@ -34,7 +36,8 @@
     YOUI_ENGINE(AutomationName.YOUI_ENGINE.toLowerCase(), YouiEngineElement.class),
     IOS_XCUI_TEST(AutomationName.IOS_XCUI_TEST.toLowerCase(), IOSElement.class),
     ANDROID_UI_AUTOMATOR(MobilePlatform.ANDROID.toLowerCase(), AndroidElement.class),
-    IOS_UI_AUTOMATION(MobilePlatform.IOS.toLowerCase(), IOSElement.class);
+    IOS_UI_AUTOMATION(MobilePlatform.IOS.toLowerCase(), IOSElement.class),
+    WINDOwS(MobilePlatform.WINDOWS, WindowsElement.class);
```

Comment:
@yodurr It is ok. According to [Java conversation rules](http://docs.oracle.com/javase/tutorial/java/javaOO/enum.html) enum elements usully have upper case name. :)
Thank you.